core/common/src/main/java/alluxio/underfs/UnderFileSystemWithLogging.java

```diff
@@ -560,6 +560,15 @@ public boolean supportsFlush() {
   }
 
   /**
+   * This is only used in the test.
```

Comment:
I've looked the code. It looks like the closest way to replace this is to use mkdirs call. I am not very familiar with this part of code. I will create a bug and add a TODO to Adit since he is the last one who made significant change to this.